src/fetch.py

```diff
@@ -423,8 +423,17 @@ def fetch(self, metadata: Path):
 if __name__ == "__main__":
     fetcher = Fetcher()
 
-    metadata_file = sys.argv[1]
-    errors = fetcher.fetch(Path(metadata_file))
+    selector = sys.argv[1]
+    errors = 0
+    if selector == "europe":
```

Comment:
We generally have ci/fetch-feeds.py for fetching multiple regions. Maybe create a new similar script for this. Having too many special cases always bears the risk of breaking some of them with every change

Reply:
So instead of adding it here, creating a python script `generate-continent-config.py` ? Would be fine by me as well.